include/openenclave/bits/log-host.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef LOG_HOST_H
+#define LOG_HOST_H
+
+#include <stdio.h>
+#include <openenclave/bits/log-common.h>
+
+int log_init(uint8_t level, const char *path);
```

Comment:
Public interfaces need to be documented. It is not clear to me what log_init with a level means.
See host.h for documentation style.